test/login-requirements.js

```diff
@@ -283,25 +235,39 @@ describe('Login', function() {
 
     const token = result.incompleteToken;
 
+    try {
+      await apos.http.post('/api/v1/@apostrophecms/login/requirement-verify', {
+        body: {
+          incompleteToken: token,
+          session: true,
+          name: 'ExtraSecret',
+          requirementValue: 'roll-off'
+        },
+        jar
+      });
+    } catch ({ status, body }) {
+      assert(status === 400);
+      assert.strictEqual(body.message, 'extra secret incorrect');
+      assert.strictEqual(body.data.requirement, 'ExtraSecret');
```

Comment:
This property is a little redundant now since the route is one requirement at a time anyway, but let's keep it for convenience.

Reply:
Oh yes it's added to the error response in getContext right? It can be removed right, as you prefer.